modules/library/src/main/scala/zio/elasticsearch/executor/HttpExecutor.scala

```diff
@@ -413,7 +413,7 @@ private[elasticsearch] final class HttpExecutor private (esConfig: ElasticConfig
 
     val sortsJson =
       if (r.sortBy.isEmpty) {
-        Obj("sort" -> Arr(ShardDoc.toJson))
+        Obj("sort" -> Arr(Json.Str(ShardDoc)), "track_total_hits" -> Bool(false))
```

Comment:
Why did you change `ShardDoc.toJson` to `Json.Str(ShardDoc)`?

Reply:
I think it's better to use already defined toJson from ElasticPrimitives.

Reply:
Forgot to change it when I was resolving a conflict 😅